tag_test.go

```diff
@@ -1503,13 +1503,13 @@ func TestEncodeBuiltinTag(t *testing.T) {
 		{
 			name: "unsigned bignum content not enclosed in expected encoding tag",
 			tag:  Tag{Number: tagNumUnsignedBignum, Content: []byte{0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00}},
-			opts: EncOptions{ByteSlice: ByteSliceExpectedEncodingBase16},
+			opts: EncOptions{ByteSlice: ByteSliceToByteStringWithExpectedConversionToBase16},
```

Comment:
It wasn't named in this PR but `ByteSliceToByteStringWithExpectedConversionToBase16` can be renamed to a shorter name.